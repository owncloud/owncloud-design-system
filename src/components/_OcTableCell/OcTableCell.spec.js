import { shallowMount } from '@vue/test-utils'
import Cell from './_OcTableCell.vue'

describe('OcTableCell', () => {
  it('Uses correct element', () => {
    const wrapper = shallowMount(Cell, {
      propsData: {
        type: 'th',
        alignH: 'right',
        alignV: 'bottom',
        width: 'shrink'
      },
      slots: {
        deafult: 'Hello world!'
      }
    })

    expect(wrapper.element.tagName).toBe('TH')
    expect(wrapper.classes()).toContain('oc-table-cell-align-right')
    expect(wrapper.classes()).toContain('oc-table-cell-align-bottom')
    expect(wrapper.classes()).toContain('oc-table-cell-width-shrink')
    expect(wrapper).toMatchSnapshot()
  })
})
