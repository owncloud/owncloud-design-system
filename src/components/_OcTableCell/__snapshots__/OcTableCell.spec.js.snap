// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OcTableCell Uses correct element 1`] = `<th class="oc-table-cell oc-table-cell-align-right oc-table-cell-align-bottom oc-table-cell-width-shrink"></th>`;
